ow_sim_tests/scripts/test_arm_find_surface.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env python
+
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+import roslib
+import unittest
+import owl_msgs.msg
+
+from action_testing import *
+from ow_lander import constants
+from geometry_msgs.msg import Vector3, Pose, Quaternion
+
+PKG = 'ow_sim_tests'
+TEST_NAME = 'arm_find_surface'
+roslib.load_manifest(PKG)
+
+class TestArmFindSurface(unittest.TestCase):
+
+  @classmethod
+  def setUpClass(cls):
+    rospy.init_node("test_arm_find_surface")
+
+    set_ignore_action_checks(True)
+    # proceed with test only when ros clock has been initialized
+    while rospy.get_time() == 0:
+      rospy.sleep(0.1)
+
+  def test_01_arm_find_surface(self):
+    arm_find_surface_result = test_arm_action(self,
+      'ArmFindSurface', owl_msgs.msg.ArmFindSurfaceAction,
+      owl_msgs.msg.ArmFindSurfaceGoal(
+        frame = 0,
+        relative = False,
+        position = Point(1.7, 0.5, constants.DEFAULT_GROUND_HEIGHT),
+        normal = Vector3(0, 0, -1),
+        distance = 0.2,
+        overdrive = 0.05,
+        force_threshold = 200,
+        torque_threshold = 200
+      ),
+      TEST_NAME, 'test_02_arm_find_surface',
+      server_timeout = 50.0 # (seconds) first action call needs longer timeout
+    )
+
+  def test_02_arm_move_cartesian(self):
+    arm_move_cartesian_result = test_arm_action(self,
+      'ArmMoveCartesian', owl_msgs.msg.ArmMoveCartesianAction,
+      owl_msgs.msg.ArmMoveCartesianGoal(
+        frame = 0,
+        relative = True,
+        pose= Pose(
+          position = Point(0, 0, 0.2),
+          orientation = Quaternion(0, 0, 0, 1)
+        )
+      ),
+      TEST_NAME, 'test_02_arm_move_cartesian'
+      # The arm_move_cartesian after every arm_find_surface is to get the arm off the ground
+    )
```

Comment:
```suggestion
   # Uisng arm_move_cartesian after every arm_find_surface to get the arm off the ground
  def test_02_arm_move_cartesian(self):
    arm_move_cartesian_result = test_arm_action(self,
      'ArmMoveCartesian', owl_msgs.msg.ArmMoveCartesianAction,
      owl_msgs.msg.ArmMoveCartesianGoal(
        frame = 0,
        relative = True,
        pose= Pose(
          position = Point(0, 0, 0.2),
          orientation = Quaternion(0, 0, 0, 1)
        )
      ),
      TEST_NAME, 'test_02_arm_move_cartesian'
    )
```

Reply:
The diff makes it hard to see, but I simply moved the comment to the top of the test unit, and did some minor rewording